crates/teloxide-core/schema.ron

```diff
@@ -329,6 +329,11 @@ Schema(
                     ty: RawTy("Recipient"),
                     descr: Doc(md: "Unique identifier for the chat where the original message was sent (or channel username in the format `@channelusername`)"),
                 ),
+                Param(
+                    name: "video_start_timestamp",
+                    ty: Option(RawTy("Seconds")),
```

Comment:
We use `Seconds` for usecases similar to `std::time::Duration`. For timestamps I think `DateTime` will fit better (see `crates/teloxide-core/src/codegen/patch.rs` for `Option(i64)` → `DateTime` patching)